docs/credentials/EcommerceInvoiceRegistrationCredential.json

```diff
@@ -0,0 +1,37 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3id.org/traceability/v1"
+  ],
+  "id": "https://example.com/123",
+  "type": [
+    "VerifiableCredential"
+  ],
+  "issuer": {
+    "id": "did:key:z6MkoTHsgNNrby8JzCNQ1iRLyW5QQ6R8Xuu6AA8igGrMVPUM"
+  },
+  "issuanceDate": "2020-03-10T04:24:12.164Z",
+  "expirationDate": "2029-03-10T04:24:12.164Z",
+  "credentialSubject": {
+    "id": "did:web:www.amazon.com",
+    "type": "EcommerceInvoiceRegistrationCredential",
+    "invoiceID": "Invoice#409",
+    "orderID": [
+      "https://example.VC_url.com/?queryID=AB06f1f744a781480c521902a1a1dbf5f1d01e7ea21daf483e7668817e58597a"
```

Comment:
this is not a safe url... it should be vc.example.com (note the `example.com` comes at the end.

Reply:
updated